crates/wysiwyg/src/dom/find_range.rs

```diff
@@ -312,19 +327,11 @@ mod test {
             find_pos(&d, &d.document_handle(), 2, 2),
             found_single_node(DomHandle::from_raw(vec![0]), 0, 2, 2, 3)
         );
-        assert_eq!(
-            find_pos(&d, &d.document_handle(), 3, 3),
-            found_single_node(DomHandle::from_raw(vec![0]), 0, 3, 3, 3)
-        );
-        assert_eq!(
-            find_pos(&d, &d.document_handle(), 3, 4),
-            found_single_node(DomHandle::from_raw(vec![1]), 3, 0, 1, 3)
-        );
-        // TODO: break up this test and name parts!
```

Comment:
Tests rearranged to remove this todo and slim this single large test down. Tests are equivalent apart from one new test.